client/store/singlePackStore.js

```diff
@@ -10,7 +10,7 @@ import axios from "axios";
 
```

Comment:
Ok i get what you are saying and i understand your code here is accomplishing the fix, but should we just move this whole slice into packs and add a new state for singlePack ? 

The main issue i have is having 2 thunks that call the same api route, and i feel like we should consolidate. Currently, the packs in packs slice is user packs, but singlePack could contain another user's pack. 

I feel like we should not have to update the entire pack to just update one pack item.